packages/suite/src/storage/index.ts

```diff
@@ -4,7 +4,7 @@ import { migrate } from './migrations';
 
 import type { SuiteDBSchema } from './definitions';
 
-const VERSION = 37; // don't forget to add migration and CHANGELOG when changing versions!
+const VERSION = 38; // don't forget to add migration and CHANGELOG when changing versions!
```

Comment:
bump not needed when there is unpublished bump already? but it doesn't matter, so let's leave it